packages/base/any/kernels/5.4-lts/configs/x86_64-all/x86_64-all.config

```diff
@@ -461,6 +461,7 @@ CONFIG_ACPI_PROCESSOR_CSTATE=y
 CONFIG_ACPI_PROCESSOR_IDLE=y
 CONFIG_ACPI_CPPC_LIB=y
 CONFIG_ACPI_PROCESSOR=y
+# CONFIG_ACPI_IPMI is not set
```

Comment:
In the same way the Linux 5.4 configuration changes, need to be split into separate commits. Please prefix them with linux-5.4.

The commit message should also contain a comparison of the bzImage size and module size of before and after the configuration change.

Reply:
OK. I would compare the size of the below files for both 5.4 and 4.19. Is it right ? thanks.
./REPO/stretch/packages/binary-amd64/onl-kernel-4.19-lts-x86-64-all_1.0.0_amd64.deb
./REPO/stretch/packages/binary-amd64/onl-kernel-4.19-lts-x86-64-all-modules_1.0.0_amd64.deb
./packages/base/amd64/kernels/kernel-4.19-lts-x86-64-all/builds/stretch/linux-4.19.81/arch/x86_64/boot/bzImage

Reply:
Yes, sounds good.